torch/csrc/autograd/profiler_kineto.cpp

```diff
@@ -85,8 +86,12 @@ struct MetadataBase {
     }
   }
 
-  void addMetadata(const std::string& key, const std::string& value) {
-    if (kineto_activity_ && !value.empty() && value != "\"\"") {
+  void addMetadata(
+      const std::string& key,
+      const std::string& value,
+      const bool no_callstack_trace = false) {
+    if (!no_callstack_trace && kineto_activity_ && !value.empty() &&
```

Comment:
`addMetadata` is a generic method. Config based omission should be handled by the caller.

Reply:
okay